boot/cmdline.go

```diff
@@ -281,6 +282,12 @@ func observeSuccessfulCommandLineCompatBoot(model *asserts.Model, m *Modeenv) (*
 	if err != nil {
 		return nil, err
 	}
+	if cmdlineExpected == "" {
+		// there is no particular command line expected for this model
+		// and system bootloader, indicating that the command line is
+		// not being tracked
+		return m, nil
+	}
```

Comment:
I think these changes in boot needs new unit tests for the new scenarios, no?

Reply:
Added a test for this too. The scenario is basically a uc20 device which does not have the trusted assets bootloader, so kernel command lines will not be populated, and we do not expect a command line for it anyway.